ui/src/services/api.js

```diff
@@ -45,3 +40,11 @@ export async function fetchTheme() {
     return { error };
   }
 }
+
+export async function fetchConfig() {
+  try {
+    return await axios.get(process.env.PUBLIC_URL + '/config.json');
+  } catch (error) {
```

Comment:
How could the line above ever `throw` something synchronously?

Reply:
Not sur understand but it throws an error when config.json is not in the configMap.